cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4363,6 +4363,14 @@
     </description>
   </property>
 
+  <property>
+    <name>task.worker.container.disk.readonly</name>
+    <value>true</value>
```

Comment:
nit: would it better to be false by default. 

Either way is fine with me.

Reply:
Since readonly will probably be the default config, I would recommend to set it to true by default.